docs/extending-documentation.md

```diff
@@ -210,76 +262,54 @@ module.exports = {
 };
 ```
 
-In the code above we have 2 places in where you can do overrides: `./.storybook/all.override.md` for generic overrides and via `getLocalOverridePath` for each component, when needed, the `rel` needs to be the same in `lion` and your own project to be able to override the right file.
-
-We insert a remarkExtend plugin instance for both the local overrides and the global overrides, and pass the necessary arguments. Then we return the new array of plugins for Mdjs.
-
-In each file you need to add a specifier which section you want to override. As a first step we want to load `lea-tab` and `lea-tab-panel` also in the project. We need to add some imports to the top of the file.
+In the code example mentioned, we have 2 places in where you can do overrides: `./.storybook/all.override.md` for generic overrides and via `getLocalOverridePath` for each component. When needed, the `rel` needs to be the same in `lion` and our own project to be able to override the right file.
 
-Use triple backtick \` to create a fenced codeblock ` ``` `, and put inside it:
+In each file we need to specify which section we want to override. We want to load `example-button` in the project:
 
-```md
+````md
+```
 ::addMdAfter(':root')
 ```
 
-Then below that, create another fenced codeblock with `js script` annotation with your imports inside it:
-
-```js
-import '../lea-tab.js';
-import '../lea-tab-panel.js';
+```js script
+import '../example-button.js';
 ```
+````
 
-And then replace each reference to `lion-tab` with `lea-tab` inside the documentation content, this also covers `lion-tab` and `lion-tab-panel`. We exclude JS snippets 'nodes', those are handled by the babel plugin already.
+And then replace each `button` with it.
 
-Create another fenced codeblock with `js ::replaceFrom(':root')` annotation:
-
-```js
+````md
+```js ::replaceFrom(':root')
 module.exports.replaceSection = node => {
-  if (node.type !== 'code' && node.value) {
+  if (node.type === 'code' && node.value) {
     let newCode = node.value;
-    newCode = newCode.replace(/lion-tab/g, 'lea-tab');
+    newCode = newCode.replace(/<lion-button/g, '<example-button');
```

Comment:
Tag replacement should have been taken care of by extend-docs and babel-extend, I think.

maybe we should also ask an external dev to 'do the tutorial'?

Reply:
That would be very nice. Got anybody in mind? Feel free to ask